src/Normalizers/Normalized/NormalizedModel.php

```diff
@@ -59,6 +56,14 @@ protected function fetchNewProperty(string $name, DataProperty $dataProperty): m
             return $this->properties[$name] = $this->model->getRelation($camelName);
         }
 
+        if (! $this->model->isRelation($name) && ! $this->model->isRelation($camelName)) {
+            try {
+                return $this->properties[$name] = $this->model->getAttribute($name);
+            } catch (MissingAttributeException) {
+                // Fallback if missing Attribute
+            }
+        }
+
```

Comment:
Not sure if we want to do this:

1) Probably 90% of the properties on a model will be regular properties without special logic, now we're checking for every property the relations, attributes, ... this is going to affect performance quite a lot

2) I hope everyone has a minimal amount of missing attributes, but always waiting on exceptions makes code slow and is something to avoid.

Reply:
Well the call to `getAttribute` was done prior to 4.9.0 (which broke this) and has never been a bottleneck

`isRelation` is not an expensive call. It does basically just 1 `isset($mutators[$key])` and one `method_exists($this, $key)` 

The exception catching is there as a bugfix. If you don't enable laravel strict mode it will never happen. But if you do, and try to access a property that doesn't exist on the model then it will throw an exception rather than do the `UnknownProperty` logic of the package which we don't want

Reply:
I'm more worried about the `$dataProperty->attributes->contains` I think we can skip that one by adding a new property to DataProperty isLoadingRelation that way we don't have to check it all the time. Once we have a cached version of that you might be right that the performance wouldn't be affected too much.

Then take a look at the comment on line 41.

If that's done this can be merged.